easycv/toolkit/blade/cv_blade_utils.py

```diff
@@ -269,14 +285,34 @@ def blade_optimize(script_model,
         summary = pd.DataFrame(results)
         logging.warning(summary.to_markdown())
 
-    output = model(*inputs)
+    print(opt_model.forward.code)
+    print(opt_model.forward.graph)
+    torch.cuda.empty_cache()
+    # warm-up
+    for k in range(10):
+        test_result = opt_model(*inputs)
+        torch.cuda.synchronize()
+
+    torch.cuda.synchronize()
     cu_prof_start()
-    if blade_config.get('enable_fp16', True):
-        with opt_blade_mixprec():
-            test_result = model(*inputs)
-    else:
+    for k in range(10):
         test_result = opt_model(*inputs)
+        torch.cuda.synchronize()
     cu_prof_stop()
-    check_results(output, test_result)
+    import torch.autograd.profiler as profiler
```

Comment:
profiler is necessary?

Reply:
we may add a config to default close this profile ,will it be allright?

Reply:
profile some times is necessary to assert whether blade optimize is OK,  for example, if all model be profiled as one torch OP group may represent a good optimize otherwise means bad optimize.

Reply:
> we may add a config to default close this profile ,will it be allright?

done